src/utils/database_pool.ts

```diff
@@ -0,0 +1,23 @@
+import { ConnectionPool, config } from 'mssql';
+
+const dbConfig: config = {
```

Comment:
Credentials would be good to move into `.env` files so that they don't show up in the git repository.